griptape/tools/email_client/tool.py

```diff
@@ -10,11 +11,64 @@
 
 @define
 class EmailClient(BaseTool):
-    host: Optional[str] = field(default=None, kw_only=True, metadata={"env": "SMTP_HOST"})
-    port: Optional[int] = field(default=None, kw_only=True, metadata={"env": "SMTP_PORT"})
-    password: Optional[str] = field(default=None, kw_only=True, metadata={"env": "SMTP_PASSWORD"})
+    smtp_host: Optional[str] = field(default=None, kw_only=True, metadata={"env": "SMTP_HOST"})
+    smtp_port: Optional[int] = field(default=None, kw_only=True, metadata={"env": "SMTP_PORT"})
+    smtp_password: Optional[str] = field(default=None, kw_only=True, metadata={"env": "SMTP_PASSWORD"})
     from_email: Optional[str] = field(default=None, kw_only=True, metadata={"env": "FROM_EMAIL"})
-    use_ssl: bool = field(default=True, kw_only=True, metadata={"env": "SMTP_USE_SSL"})
+    smtp_use_ssl: bool = field(default=True, kw_only=True, metadata={"env": "SMTP_USE_SSL"})
+
+    imap_url: Optional[str] = field(default=None, kw_only=True, metadata={"env": "IMAP_URL"})
+    imap_user: Optional[str] = field(default=None, kw_only=True, metadata={"env": "IMAP_USER"})
```

Comment:
`imap_email` for consistency?

Reply:
could be different from email even if rare